diff -u a/minimalist.pl b/minimalist.pl
--- a/minimalist.pl	2005-06-17 12:02:43.000000000 +0200
+++ b/minimalist.pl	2005-06-27 01:02:49.000000000 +0200
@@ -30,7 +30,7 @@
 use integer;
 
 $version = '2.5(3) (Good Bowl)';
-$config = "/usr/local/etc/minimalist.conf";
+$config = "/etc/mail/minimalist/minimalist.conf";
 
 # Program name and arguments for launching if commands in message's body
 $running = $0." --body-controlled ".join ($", @ARGV);
